redis/redis.go

```diff
@@ -131,3 +131,12 @@ func NewClient(options Options) (Client, error) {
 
 	return result, nil
 }
+
+// NewWithClient method creates a new `gokv.redis.Client` with given `redis.Client`.
+// redis related options are ignored, only Codec is applicable
```

Comment:
Maybe mention that the client is expected to be fully configured, with an address of a Redis server/cluster etc.

Reply:
done